main/src/cgeo/geocaching/EditWaypointActivity.java

```diff
@@ -164,8 +165,11 @@ protected void onPostExecute(final Geocache cache) {
                 if (own) {
                     initializeWaypointTypeSelector();
                 } else {
-                    waypointName.setEnabled(false);
-                    note.setEnabled(false);
+                    waypointName.setInputType(EditorInfo.TYPE_NULL);
+                    waypointName.setTextIsSelectable(true);
+
+                    note.setInputType(EditorInfo.TYPE_NULL);
+                    note.setTextIsSelectable(true);
```

Comment:
Same for these 2 lines. We now have 2 times 3 lines of identical code, so extracting a method seems reasonable.